examples/demo-rollup/stf/src/hooks_impl.rs

```diff
@@ -13,24 +13,34 @@ use crate::runtime::Runtime;
 
 impl<C: Context, Da: DaSpec> TxHooks for Runtime<C, Da> {
     type Context = C;
+    type PreArg = u64;
+    type PreResult = C;
```

Comment:
Let's construct Context outside the `pre_dispatch_tx_hook` as before. I will need to update the code and it will be easier this way.

Reply:
Done!